src/Integration.Vsix/Suppression/LiveIssue.cs

```diff
@@ -29,8 +29,8 @@ namespace SonarLint.VisualStudio.Integration.Vsix.Suppression
     /// </summary>
     internal class LiveIssue
     {
-        public LiveIssue(Diagnostic diagnostic, string issueFilePath, string projectGuid, int startLine,
-            string wholeLineText)
+        public LiveIssue(Diagnostic diagnostic, string projectGuid, string issueFilePath = "", int startLine = 0,
```

Comment:
Does the rule about not using default values only apply to public methods in public classes?

Reply:
I think we removed it from default profile with @valhristov a couple of months ago because it makes sense only when you do interop/cross-language dev.